### PR TITLE
[DOCS-8992] Add helm chart values instructions for connecting Agent to Worker

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

- Adds tabs for Agent configuration and helm chart values instructions.
- Adds helm chart values instructions.

[DOCS-8992](https://datadoghq.atlassian.net/browse/DOCS-8992)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-8992]: https://datadoghq.atlassian.net/browse/DOCS-8992?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
     url: "http://<OPW_HOST>:8282"
 ```
 
-`<OPW_HOST>` is the IP/URL of the host (or load balancer) associated with the Observability Pipelines Worker. For CloudFormation installs, the `LoadBalancerDNS` CloudFormation output has the correct URL to use. For Kubernetes installs, the internal DNS record of the Observability Pipelines Worker service can be used. For example: `datadog-observability-pipelines-worker.default.svc.cluster.local`.
+`<OPW_HOST>` is the IP/URL of the host (or load balancer) associated with the Observability Pipelines Worker.
 
-At this point, your observability data should be going to the Worker, processed by the pipeline, and delivered to Datadog.
+For CloudFormation installs, use the `LoadBalancerDNS` CloudFormation output for the URL.
+
+For Kubernetes installs, the internal DNS record of the Observability Pipelines Worker service can be used. For example: `opw-observability-pipelines-worker.default.svc.cluster.local`.
```
Comment (cswatt) [inline layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent.md:13]:
```suggestion
For Kubernetes installs, you can use the internal DNS record of the Observability Pipelines Worker service. For example: `opw-observability-pipelines-worker.default.svc.cluster.local`.
```

File: layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent_kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+To send Datadog Agent logs to the Observability Pipelines Worker, update your Datadog helm chart [values.yaml][1021] with the following environment variables. See [Agent Environment Variables][1022] for more information.
```
Comment (cswatt) [inline layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent_kubernetes.md:1]:
```suggestion
To send Datadog Agent logs to the Observability Pipelines Worker, update your Datadog Helm chart [`datadog-values.yaml`][1021] with the following environment variables. See [Agent Environment Variables][1022] for more information.
```

More recently, that file is referred to as `datadog-values.yaml` (previously we just called it `values.yaml`)

File: layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent_kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  env:
+    - name: DD_OBSERVABILITY_PIPELINES_WORKER_LOGS_ENABLED
+      value: true
+    - name: DD_OBSERVABILITY_PIPELINES_WORKER_LOGS_URL
+      value: "http://<OPW_HOST>:8282"
+```
+
+`<OPW_HOST>` is the IP/URL of the host (or load balancer) associated with the Observability Pipelines Worker.
+
+For Kubernetes installs, the internal DNS record of the Observability Pipelines Worker service can be used. For example: `opw-observability-pipelines-worker.default.svc.cluster.local`.
```
Comment (cswatt) [inline layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent_kubernetes.md:14]:
```suggestion
For Kubernetes installs, you can use the internal DNS record of the Observability Pipelines Worker service. For example: `opw-observability-pipelines-worker.default.svc.cluster.local`.
```

**Review by cswatt — COMMENTED** [review]
Looks good, some small feedback!

**Review by ckelner — APPROVED** [review]
LGTM!

File: layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent_kubernetes.md (hunk 2 above)
Comment (maycmlee) [reply layouts/shortcodes/observability_pipelines/log_source_configuration/datadog_agent_kubernetes.md:1]: Good to know!

**Review by cswatt — APPROVED** [review]

**Pull request merged**